IntelliNode/package.json

```diff
@@ -29,7 +29,9 @@
   "homepage": "https://intellinode.ai",
   "dependencies": {
     "axios": "^1.3.6",
+    "dotenv": "^16.0.3",
     "form-data": "^4.0.0",
-    "dotenv": "^16.0.3"
+    "tiktoken": "^1.0.10",
```

Comment:
avoiding adding new library to intellinode we want the module to be light.

Reply:
ok